products/extensions/b2b-components/employee-management/concept/api-route-restriction-for-employees.md

```diff
@@ -0,0 +1,31 @@
+# API Route Restriction for Employees
+
+## General
```

Comment:
```suggestion
```
Think we can remove this. Text at this place will be general afterall.

Alternatively, I think "Overview" will be a better fit.